src/Std/Time/Date/Unit/Day.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Time
+import Std.Time.Internal
+import Lean.Data.Rat
```

Comment:
CI is failing because it doesn't find `Lean.Data.Rat`. From my understanding the dependency chain should go `Init` imported by `Std` imported by `Lean` so this `import` is not correct. If we want to use it in `Std` we probably need to move `Lean.Data.Rat` to `Std.Data.Rat`.